data/styles/clear/include/Icons.mapcss

```diff
@@ -3010,50 +3010,65 @@ area|z15-[amenity=sanitary_dump_station]
 
 /* 7.1 Parking */
 
-node|z16[amenity=parking],
-area|z16[amenity=parking]
-{icon-image: parking-s.svg;icon-min-distance: 25;}
-node|z16[amenity=parking][access=permissive],
-node|z16[amenity=parking][access=private]
-{icon-image: zero-icon.svg;}
-
-node|z17[amenity=parking],
-area|z17[amenity=parking]
-{icon-image: parking-m.svg;}
-node|z17[amenity=parking][access=permissive],
-node|z17[amenity=parking][access=private]
-{icon-image: zero-icon.svg;}
-
+node|z16-[amenity=parking],
+area|z16-[amenity=parking],
+node|z16-[amenity=parking][location=underground],
+area|z16-[amenity=parking][location=underground]
+{icon-image: parking-m.svg;icon-min-distance: 5;}
+node|z18-[amenity=parking][location=underground],
+area|z18-[amenity=parking][location=underground],
 node|z18-[amenity=parking],
 area|z18-[amenity=parking]
-{icon-image: parking-m.svg;font-size: 11;}
+{font-size: 11;}
+
+node|z16-[amenity=parking][access=private],
+area|z16-[amenity=parking][access=private],
+/* =no doesn't work in kothic */
+/* node|z16-[amenity=parking][access=no], */
+/* area|z16-[amenity=parking][access=no], */
+node|z17-[amenity=parking_entrance][access=permissive],
+node|z17-[amenity=parking_entrance][access=private]
+{icon-image: ;}
```

Comment:
`[amenity=parking_entrance][access=permissive]` is defined later in the file, so there is nothing to override at this point;

in fact the later declaration will override this one and those entrances will have an icon

Reply:
good catch, fixed 👍